custom_typings/chalk.d.ts

```diff
@@ -0,0 +1,25 @@
+declare namespace Chalk {
```

Comment:
It does but unfortunately it doesn't support the `dim` feature (older version) and seems to be written in a way that can't be easily extended. I'll add a comment and maybe after your PR is merged we can update the DT version.